core/src/server/http/http_response.cpp

```diff
@@ -7,15 +7,14 @@
 
```

Comment:
looks great! thank you
Do you want to fix/add anything or I can merge it?

Reply:
> looks great! thank you Do you want to fix/add anything or I can merge it?

Happy to hear that! I don't plan to change anything in this PR, so you can merge it